src/js/lib/Scrollspy.jsx

```diff
@@ -142,36 +144,42 @@ export class Scrollspy extends React.Component {
   }
 
   _handleSpy () {
-    let timer
```

Comment:
I removed this as it appears to be dead code. Each time you enter the `_handleSpy` function `timer` will be set to `undefined` meaning the if branch will never execute.